### PR TITLE
Reader Manage Following: add search followed sites UI

### PR DESCRIPTION
**Pull request by bluefuton**
Adds the search box for searching existing feeds:

1754d3c0-1e08-11e7-87ad-e0b05c791eb2

Part of #12960.

### CODE REVIEW

**Comment by bluefuton** [comment]
Todo:
- [ ] Remove magnify icon from search input
- [ ] Do we need SearchCard or can we use straight-up `components/search`?

File: client/reader/following-manage/search-followed.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		return (
+			<SearchCard
+				compact={ true }
+				isOpen={ true }
+				pinned={ false }
+				key="searchFollowed"
+				autoFocus={ false }
+				className="following-manage__search-followed"
+				additionalClasses="following-manage__search-followed-input"
+				placeholder={ this.props.translate( 'Search Followed Sites' ) }
```
Comment (a8ci18n) [inline client/reader/following-manage/search-followed.jsx]: Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Search your followed sites' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/following-manage-search-following-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/reader/following-manage/search-followed.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		return (
+			<SearchCard
+				compact={ true }
+				isOpen={ true }
+				pinned={ false }
+				key="searchFollowed"
+				autoFocus={ false }
+				className="following-manage__search-followed"
+				additionalClasses="following-manage__search-followed-input"
+				placeholder={ this.props.translate( 'Search Followed Sites…' ) }
```
Comment (a8ci18n) [inline client/reader/following-manage/search-followed.jsx]: Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Search your followed sites' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/following-manage-search-following-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (samouri) [inline client/reader/following-manage/search-followed.jsx]: @bluefuton ^ ?

File: client/reader/following-manage/index.jsx [hunk 3]
```diff
@@ -115,6 +115,7 @@ class FollowingManage extends Component {
 							delaySearch={ true }
 							delayTimeout={ 500 }
 							placeholder={ searchPlaceholderText }
+							additionalClasses="following-manage__search-new"
```
Comment (samouri) [inline client/reader/following-manage/index.jsx]: whats this class for?

File: client/reader/following-manage/search-followed.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+		onSearch: noop,
+	}
+
+	render() {
+		return (
+			<SearchCard
+				compact={ true }
+				isOpen={ true }
+				pinned={ false }
+				key="searchFollowed"
```
Comment (samouri) [inline client/reader/following-manage/search-followed.jsx]: is the key doing anything? I think keys are usually just for when there is more of one

File: client/reader/following-manage/search-followed.jsx (hunk 2 above)
Comment (bluefuton) [inline client/reader/following-manage/search-followed.jsx]: I considered it, but I don't think the translated string is short enough to be visible as a placeholder in the input.

File: client/reader/following-manage/index.jsx (hunk 3 above)
Comment (bluefuton) [inline client/reader/following-manage/index.jsx]: Nope, but there are two search boxes on the page: one for new subs, and one for existing subs, so I thought it was important to give them a different name.

Originally the styles for this input were targeted at `.following-manage .search`, which targets both search boxes.

File: client/reader/following-manage/search-followed.jsx (hunk 4 above)
Comment (bluefuton) [inline client/reader/following-manage/search-followed.jsx]: True. Also removed the `ref` which was a leftover from using the old URLSearch mixin.

File: client/reader/following-manage/index.jsx (hunk 3 above)
Comment (samouri) [inline client/reader/following-manage/index.jsx]: ah. so the "new" here == "new subscriptions"

File: client/reader/following-manage/search-followed.jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+	render() {
+		return (
+			<SearchCard
+				compact={ true }
+				isOpen={ true }
+				pinned={ false }
+				autoFocus={ false }
+				className="following-manage__search-followed"
+				additionalClasses="following-manage__search-followed-input"
+				placeholder={ this.props.translate( 'Search Followed Sites…' ) }
```
Comment (a8ci18n) [inline client/reader/following-manage/search-followed.jsx]: Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Search your followed sites' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/following-manage-search-following-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by samouri** [comment]
giving this a test run

**Pull request merged**